pkg/core/models.go

```diff
@@ -455,3 +462,12 @@ type TestExecutionArgs struct {
 	FrameWorkVersion  int
 	CWD               string
 }
+
+// unique string to identify location of test
+type LocatorConfig struct {
+	Locator string `json:"locator"`
+}
+
+type InputLocatorConfig struct {
```

Comment:
exported member should have defined comments. Also, can you check why these are not detected by golangci-lint gh-action check?

Reply:
This struct check is done by golint, which is disabled in golangci-lint by default. To have this check we need to enable golint in the golangci-lint YAML config file.